src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__product">
+        <img 
+          class="card__photo-imac" 
+          src="./images/imac.jpeg" 
+          alt="iMac"
+        >
+      </div>
+
+      <div class="card__about">
+        <p class="card__about-product">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__about-code">Product code: 195434</p>
+      </div>
+
+      <div class="card__reviews">
+        <p class="card__reviews-count">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-count">$2,199</p>
+      </div>
+
+      <div class="card__button">
+        <a 
+          class="card__button-link" 
+          data-qa="hover" 
+          type="submit" 
+          href="/html"
+        >
+          Buy
+        </a>
+      </div>
+    </div>
+
+    <div class="stars">
```

Comment:
implement this point (check requirements and tips)
<img width="868" alt="image" src="https://user-images.githubusercontent.com/102411130/203506628-a986cc04-4b41-4b3f-a7e7-7a272ea834e8.png">